policies/decoding-image-default-sync.md

```diff
@@ -0,0 +1,60 @@
+`decoding-image-default-sync` Policy
+===========
+
+The `decoding-image-default-sync` policy-controlled feature can be used for a main frame or a nested `<iframe>` to override the default behavior of the [`decoding` attribute](https://html.spec.whatwg.org/multipage/embedded-content.html#attr-img-decoding) for images.
+
+What does that mean?
+------------
+With the `decoding-image-default-sync` policy, developers could prioritize the decoding of `<img>`s on a web page.
+Developers may use the `decoding` attribute to specify the decoding method for an image, this however could become a cumbersome process and not quite scalable for larger web sites specially given that applying the attribute is origin-agnostic. The proposed policy aims to resolve this issue by changing a browser's default decision for `decoding` behavior.
+
+Proposed Solution
+------------
+The `decoding` attribute can take one of the three values:
+  * **`sync`** indicating that the image should decode synchronously.
+  * **`auto`** the browser makes the call for the decoding behavior; it probably defaults to `sync`.
```

Comment:
> **`auto`** [...] it probably defaults to `sync`.

I'm not sure if this is true, at least for Firefox this suggests otherwise: https://github.com/w3c/csswg-drafts/issues/2095#issuecomment-465309263.